lib/core/controlplane.rb

```diff
@@ -25,13 +25,15 @@ def profile_exists?(profile)
   def profile_create(profile, token)
     sensitive_data_pattern = /(?<=--token )(\S+)/
     cmd = "cpln profile create #{profile} --token #{token}"
-    cmd += " > /dev/null" if Shell.should_hide_output?
+    cmd += " > /dev/null" if Shell.should_hide_output? || ENV["RAILS_ENV"] == "test"
+    cmd += " 2>&1" if ENV["RAILS_ENV"] == "test"
```

Comment:
let's dry this up and make more semantic

```ruby
    cmd += " > /dev/null" if Shell.should_hide_output? || ENV["RAILS_ENV"] == "test"
    cmd += " 2>&1" if ENV["RAILS_ENV"] == "test"
```

copilot says:

You can create a helper method that builds the command based on the environment and verbosity. This helper method can be used in both `run_show_only_stderr_unless_verbose` and `run_always_show_output` methods. Here's how you can do it:

```ruby
def build_command(cmd, always_show_output: false)
  if ENV["RAILS_ENV"] == "test"
    cmd += " > /dev/null 2>&1"
  elsif !always_show_output && Shell.should_hide_output?
    cmd += " > /dev/null"
  end
  cmd
end

def run_show_only_stderr_unless_verbose(cmd)
  cmd = build_command(cmd)
  Kernel.exec(cmd)
end

def run_always_show_output(cmd)
  cmd = build_command(cmd, always_show_output: true)
  Kernel.exec(cmd)
end
```

In this refactored code, the `build_command` method takes care of appending the necessary redirections to the command based on the environment and verbosity. The `run_show_only_stderr_unless_verbose` and `run_always_show_output` methods then use this helper method to build the command before executing it.